utils/ifupdown.sh.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
> This script is intended to be used by ifupdown, which is a Debian/Ubuntu specific package

A buildsystem such as buildroot provides an ifupdown package which doesn't require Bash.

> Also note that bridge-stp (which is more likely to be used on an embedded system) already contains several bashisms and uses #!/bin/bash

You are correct, this was on my todo list of stuffs to fix. I'll open a ticket.